### ISSUE TITLE
Json serialization plugin based on System.Text.Json

### ISSUE THREAD

**Issue by zarusz** [zarusz/SlimMessageBus#30]
The [System.Text.Json](https://www.nuget.org/packages/System.Text.Json) is gaining more popularity.

We need add serialization support for that library.

**Comment by baoduy** [comment]
I have already developed this one as below. If it is ok I could be able to submit a PR as well.

```csharp
public class JsonMessageSerializer: IMessageSerializer
{
    public JsonSerializerOptions Options { get; set; } = new(JsonSerializerDefaults.Web)
    {
        DefaultIgnoreCondition = JsonIgnoreCondition.WhenWritingNull,
        AllowTrailingCommas = true
    };
    
    public byte[] Serialize(Type t, object message) => JsonSerializer.SerializeToUtf8Bytes(message, t, Options);

    public object Deserialize(Type t, byte[] payload) => JsonSerializer.Deserialize(payload, t, Options)!;
}
```

**Comment by baoduy** [comment]
@zarusz I just submitted a PR for this. Could you help to review it?

Thanks
Steven

**Comment by zarusz** [comment]
@baoduy , yes I will sometime over the weekend! :)

Do you think the NativeJson name is okay? I am slightly hesitant - need to see how other projects/libraries call these plugins for the new JSON library.

**Comment by baoduy** [comment]
Yah feel free to suggest the name 😊

**Comment by zarusz** [comment]
This has been implemented as part of the PR #118  by @baoduy
